pkg/controllers/vdb/revivedb_reconciler.go

```diff
@@ -96,6 +101,18 @@ func (r *ReviveDBReconciler) Reconcile(ctx context.Context, _ *ctrl.Request) (ct
 	return g.checkAndRunInit(ctx)
 }
 
+func (r *ReviveDBReconciler) isRestoreSupported() error {
+	if vinf, ok := r.Vdb.MakeVersionInfo(); !vmeta.UseVClusterOps(r.Vdb.Annotations) ||
```

Comment:
Let's use `MakeVersionInfoCheck`. That way if we can't find the version in the vdb, we will return a specific error, rather than the restore error below.